src/azure-cli/azure/cli/command_modules/profile/_help.py

```diff
@@ -11,9 +11,9 @@
 type: command
 short-summary: Log in to Azure.
 long-summary: >-
-    By default, this command logs in with a user account. CLI will try to launch a web browser to log in interactively.
-    If a web browser is not available, CLI will fall back to device code login.
-
+    By default, this command logs in with a user account.
+    Windows systems use Web Account Manager (WAM), and Linux and macOS systems use browser-based login by default.
+    If WAM or a web browser is not available, CLI will fall back to device code login.
```

Comment:
This is to address https://github.com/Azure/azure-cli/pull/28841#issuecomment-2106710296.

I copied the text from https://learn.microsoft.com/en-us/cli/azure/authenticate-azure-cli-interactively#interactive-login

Perhaps we can use

```suggestion
    CLI uses Web Account Manager (WAM) on Windows, and browser-based login on Linux and macOS by default.
    If WAM or a web browser is not available, CLI will fall back to device code login.
```

Reply:
One thing, @jiasli, there are many "CLIs" at Microsoft.  Granted, if a customer is receiving this message in-line, we are obviously talking about "Azure CLI", but just letting you know that in docs, I always use the full name. I also have a quality remediation work item to change this in TOCs and other docs that talk about our product. No update requested.  Just communicating.

Reply:
I have changed "CLI" to "Azure CLI" to be explicit.

@dbradish-microsoft, which statement do you prefer?

> Windows systems use Web Account Manager (WAM), and Linux and macOS systems use browser-based login by default. If WAM or a web browser is not available, CLI will fall back to device code login.

or 

> Azure CLI uses Web Account Manager (WAM) on Windows, and browser-based login on Linux and macOS by default. If WAM or a web browser is not available, Azure CLI will fall back to device code login.

Reply:
@jiasli, definitely the 2nd sentence.